tasmoadmin/rootfs/etc/cont-init.d/ingress.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/with-contenv bashio
```

Comment:
Ingress is not an application. Please split across the `nginx.sh` and `tasmoadmin.sh`